Plate_2_data/4.processing_features/scripts/feature_select_sc_plate2.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pandas as pd
+import pathlib
+
+from pycytominer import feature_select
+from pycytominer.cyto_utils import output
+
+import sys
+sys.path.append("../../utils")
+import sc_extraction_utils as sc_utils
+
+
+# In[2]:
+
+
+# output directory for annotated file
+output_dir = pathlib.Path("./data/")
+
+# dictionary with each run for the cell type
+run_info_dictionary = {
+    "SHSY5Y_first_run": {
+        # path to parquet file from annotate function
+        "normalized_path": str(pathlib.Path("./data/SHSY5Y_first_run_sc_norm.parquet"))
+    },
+    "SHSY5Y_second_run": {
+        # path to parquet file from annotate function
+        "normalized_path": str(pathlib.Path("./data/SHSY5Y_second_run_sc_norm.parquet")),
+    },
+}
+
+
+# In[3]:
+
+
+# list of operations for feature select function to use on input profile
+feature_select_ops = [
+    "variance_threshold",
+    "correlation_threshold",
+    "blocklist",
+]
```

Comment:
Consider adding a small comment as to why these feature select functions were chosen.

Reply:
This is a good comment, I actually haven't gone deeper into why these operations are chosen other than these are the standard ones I have seen in pipelines with CellProfiler feature data.

I will start investigating this and add this into another PR. Thank you for bringing this up!